plugins/services/src/js/components/forms/MultiContainerVolumesFormSection.js

```diff
@@ -94,6 +94,10 @@ class MultiContainerVolumesFormSection extends Component {
         path: "volumeMounts"
       });
 
+      if (volumes.type === VolumeConstants.type.unknown) {
+        return <noscript />;
```

Comment:
Thank you for noticing! Somehow I thought we still use an older React, I had a rough day... 😅